### ISSUE TITLE
java.lang.VerifyError: Expecting a stackmap frame at branch target

### ISSUE THREAD

**Issue by plul** [licel/jcardsim#142]
On branch master, I can successfully perform `mvn initialize` and `mvn clean install` when pointing at any one of jcdk 3.0.5u1/3.0.5u2/3.0.5u3 in `pom.xml`. (I can't tell which is correct btw).

However, at runtime when I try to create a `CardSimulator`, I get the following error:

```
Exception in thread "main" java.lang.VerifyError: Expecting a stackmap frame at branch target 13
Exception Details:
  Location:
    javacard/framework/APDU.setOutgoingNoChaining()S @6: ifeq
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: 2ab4 005f 0333 9900 0704 b800 8f2a b400
    0x0000010: 5f03 0454 2ab4 005f 0504 542a b400 5d08
    0x0000020: 0656 2ab4 005d 0335 ac                 
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getDeclaredConstructors(Class.java:2020)
	at com.licel.jcardsim.base.SimulatorRuntime.<init>(SimulatorRuntime.java:82)
	at com.licel.jcardsim.base.SimulatorRuntime.<init>(SimulatorRuntime.java:75)
	at com.licel.jcardsim.smartcardio.CardSimulator.<init>(CardSimulator.java:44)
(...)
```

I tried to check out some old commit such as `a66e111` from two and a half years ago. There,  everything works fine (compiling against jcdk 3.0.4). The above error does not happen in this case.

**Comment by skrisztian** [comment]
I have exactly the same issue. However, this helped (see the Troubleshooting section):
https://github.com/ph4r05/javacard-gradle-template

**Comment by ph4r05** [comment]
The culprit is fixed in #148 and #149
